vividus-plugin-azure-resource-manager/src/main/java/org/vividus/azure/resourcemanager/ResourceManagementSteps.java

```diff
@@ -54,6 +55,31 @@ public ResourceManagementSteps(AzureProfile azureProfile, TokenCredential tokenC
         this.azureProfile = azureProfile;
     }
 
+    /**
+     * Gets the info about the specified Azure resource using the declared Azure resource URL and saves it to a
+     * variable. For more information, see the
+     * <a href="https://docs.microsoft.com/en-us/rest/api/azure/">Azure REST API reference</a>.
+     *
+     * @param azureResourceUrl Azure resource REST API URL
```

Comment:
plese make Javadoc very similar to regular docs

Reply:
Made them identical.